src/storage/src/chunk.rs

```diff
@@ -160,6 +172,20 @@ impl ChunkReaderBuilder {
 
         Ok(ChunkReaderImpl::new(schema, Box::new(reader)))
     }
+
+    /// Build time range predicate from schema and filters.
+    pub fn build_time_range_predicate(&self) -> TimestampRange {
+        let Some(ts_col) = self.schema.user_schema().timestamp_column() else { return TimestampRange::min_to_max() };
+        TimeRangePredicateBuilder::new(&ts_col.name, &self.filters).build()
+    }
+
+    /// Check if SST file's time range matches predicate.
+    pub fn file_in_range(file: &FileHandle, predicate: TimestampRange) -> bool {
```

Comment:
If the predicate is `min_max` range, we can return true directly as a fast path.